ios/FluentUI.Demo/FluentUI.Demo/Demos/HUDDemoController.swift

```diff
@@ -148,4 +151,117 @@ class HUDDemoController: DemoController {
             }
         }
     }
+
+    override func showMessage(_ message: String, autoDismiss: Bool = true, completion: (() -> Void)? = nil) {
+        let alert = UIAlertController(title: message, message: nil, preferredStyle: .alert)
+        present(alert, animated: true)
+
+        if autoDismiss {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+                self.dismiss(animated: true)
+            }
+        } else {
+            let okAction = UIAlertAction(title: "OK", style: .default) { _ in
+                self.dismiss(animated: true, completion: completion)
+            }
+            let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+            alert.addAction(okAction)
+            alert.addAction(cancelAction)
+        }
+
+    }
+
+    private enum HUDDemoSection: CaseIterable {
+        case swiftUI
+        case activity
+        case success
+        case failure
+        case custom
+        case nonblocking
+        case nolabel
+        case tapgesture
+        case updatingcaption
+
+        var isDemoSection: Bool {
+            return self != .swiftUI
+        }
+
+        var rows: [HUDDemoRow] {
+            switch self {
+            case .swiftUI:
+                return [.swiftUIDemo]
+            case .activity,
+                 .success,
+                 .failure,
+                 .custom,
+                 .nonblocking,
+                 .nolabel,
+                 .tapgesture,
+                 .updatingcaption:
+                return [.showHUD]
+            }
+        }
+
+        var action: Selector {
```

Comment:
Nice! I don't think we've used this pattern before in the demo app, but I'm a big fan for cases where we want UIKit-style dynamic dispatch.